<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgLineCounterClass</class>
 <widget class="QDialog" name="DlgLineCounterClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>223</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LineCounter(Scan)-拖拽文件或文件夹</string>
  </property>
  <property name="windowIcon">
   <iconset resource="DlgLineCounter.qrc">
    <normaloff>:/DlgLineCounter/Addin_LineCounter.ico</normaloff>:/DlgLineCounter/Addin_LineCounter.ico</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QProgressBar" name="m_progressBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>441</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>451</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="m_lblFileCount">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>文件数</string>
    </property>
   </widget>
   <widget class="QLabel" name="m_lblFilePath">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>文件路径</string>
    </property>
   </widget>
   <widget class="QLabel" name="m_lblExt">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>后缀名</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="m_edtFileCount">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="m_edtFilePath">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>361</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="m_edtExt">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>361</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="m_edtLineCount">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="m_lblLineCount">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>行数</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="m_lblIteredFile">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>441</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="DlgLineCounter.qrc"/>
 </resources>
 <connections/>
</ui>
